index.bs

```diff
@@ -3304,11 +3274,11 @@ The sample code for generating and registering a new key follows:
       pubKeyCredParams: [
         {
           type: "public-key",
-          alg: "ES256",
+          alg: -7 // "ES256" as registered in the IANA COSE Algorithms registry
```

Comment:
we need refs for IANA COSE Algorithms registry and for JOSE Algs registry. I'm working on adding those.